paddle/fluid/imperative/prepared_operator.cc

```diff
@@ -44,7 +45,7 @@ static void PrepareData(const platform::Place& place,
 
         // TODO(jiabin): Support transform data layout when we Verify it on more
         // tests
-        if (!(tmp_place == place)) {
+        if (!(tmp_place == place) || true) {
```

Comment:
Why should this always be true?

Reply:
Previously I have removed the if completely but then it didn't pass the Coverage CI.
I hope @JiabinYang could review this because of the comment above that seems important to have a look at this. What kind of tests does he have in mind.